WORKSPACE

```diff
@@ -1,5 +1,7 @@
+workspace(name = "com_google_glog")
```

Comment:
This is also unnecessary because you don't have multiple workspaces

Reply:
Done.